src/components/App.js

```diff
@@ -4,15 +4,16 @@ import '../css/App.css'
 import Nav from './Nav'
 import PosterGrid from './PosterGrid'
 import MovieDetails from './MovieDetails'
-import movieData from '../movieData'
+import endpoints from '../endpoints'
```

Comment:
It's nice to see these two imports right next to each other.  This really helps to solidify the idea that we are replacing the data file with data from different endpoints, which are contained in an object.